ext/ldap/ldap.stub.php

```diff
@@ -4,177 +4,177 @@
 
 #ifdef HAVE_ORALDAP
 /** @return resource|false */
-function ldap_connect(string $hostname = UNKNOWN, int $port = 389, string $wallet = UNKNOWN, string $wallet_passwd = UNKNOWN, int $authmode = GSLC_SSL_NO_AUTH) {}
+function ldap_connect(?string $uri = null, int $port = 389, string $wallet = UNKNOWN, string $wallet_passwd = UNKNOWN, int $authmode = GSLC_SSL_NO_AUTH) {}
 #else
 /** @return resource|false */
-function ldap_connect(string $hostname = UNKNOWN, int $port = 389) {}
+function ldap_connect(?string $uri = null, int $port = 389) {}
 #endif
 
-/** @param resource $link_identifier */
-function ldap_unbind($link_identifier): bool {}
+/** @param resource $link */
+function ldap_unbind($link): bool {}
```

Comment:
Over in #6172 we're changing `$mysql_link` to `$connection`.  It's probably wise to do the same here.  I don't know anyone anywhere that calls this thing a "link".

Reply:
Hum, it was called $link in some functions and $link_identifier in others which is why I changed to use $link everywhere.

I usually call this $ldap, I think I’ve seen $cid for connection id.

I’m not sure $connection is better, it’s way longer for sure (and for me prone to errors since we write "connexion" in french).

Are there other occurencies than mysql?
Can other people give their opinion (especially native english speakers)?

Reply:
hello @MCMic i'am also in favour of $ldap i think it make more sense in the context of a ldap library

Cheers